deploy/jenkins-ci/Makefile.operator.jenkins

```diff
@@ -156,8 +162,12 @@ endif
 operator-prepare-master-next-version:
 	# Only minor releases require to prepare the master branch for the next release
 ifeq ($(RELEASE_TYPE),minor)
+	OPERATOR_IMAGE_VERSION="latest" \
+	  $(MAKE) generate-all-in-one
+	cp _output/kiali-operator-all-in-one.yaml deploy/
 	sed -i -r "s/^VERSION \?= (.*)/VERSION \?= v$(OPERATOR_BUMPED_VERSION)-SNAPSHOT/" Makefile
 	git add Makefile
+	git add deploy/kiali-operator-all-in-one.yaml
```

Comment:
There is the unfortunate outcome that the all-in-one YAML file will be updated on master only when the operator is released. Not sure if you would prefer to update it "automagically" every time is needed.

Just a comment. Not sure how we really could deal with this.

Reply:
Yes, this is the intention. We auto-generate this all-in-one when the operator is released. If we ever need to update it any other time (for whatever reason) we can always manually generate it. The point here is that we release an all-in-one for each minor release, and we just maintain a copy in master. But the master copy is less important than the ones in the release branch.